gnucash/report/test/test-commodity-utils.scm

```diff
@@ -655,15 +655,15 @@
                           (gnc-dmy2time64-neutral 20 02 2016)
                           #f #f)))
         (test-equal "gnc:case-exchange-time-fn weighted-average 20/02/2012"
-          307/5
+          0
```

Comment:
The commit message needs to explain why the answers changed, otherwise it looks like you've done something wrong and patched the test to hide it.

Reply:
I'll amend when rewriting the commit message.

Reply:
test fix commit message amended.